server/v2/api/rest/config.go

```diff
@@ -0,0 +1,18 @@
+package rest
+
+func DefaultConfig() *Config {
+	return &Config{
+		Enable:  true,
+		Address: "localhost:8080",
+	}
+}
+
+type CfgOption func(*Config)
```

Comment:
There's no option defined, so let's delete this.

Reply:
but is the only way to change options. You mean the aliasing? Check server_test

Reply:
True, could you define this one then: 

```go
// OverwriteDefaultConfig overwrites the default config with the new config.
func OverwriteDefaultConfig(newCfg *Config) CfgOption {
	return func(cfg *Config) {
		*cfg = *newCfg
	}
}
```